opentelemetry-api/src/opentelemetry/propagate/__init__.py

```diff
@@ -109,8 +109,9 @@ def inject(
     """Uses the configured propagator to inject a Context into the carrier.
 
     Args:
-        carrier: An object that contains a representation of HTTP
-            headers. Should be paired with setter, which
+        carrier: An object that can carry information including 
+            the traceparent id.
```

Comment:
This interface is not tied to traceparent or any other standard. We might consider just copying the defintion from the spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#carrier

Does that definition read OK to you?

Reply:
Yeah, perfect, I'll sort it now